hack/install-wrapper.sh

```diff
@@ -184,13 +188,7 @@ function download_antrea_files() {
     fi
     download_file "${ANTREA_INSTALL_SCRIPT}" "${tmp_dir}"/${INSTALL_SCRIPT}
     download_file "${ANTREA_CONFIG}" "${tmp_dir}"/${ANTREA_AGENT_CONF}
-    # TODO: Cleanup post antrea 1.9 release.
-    if [ -f "$AGENT_BIN" ]; then
-        # Temporary solution to use antrea-agent binary from local.
-        cp "$AGENT_BIN" "${tmp_dir}"/${ANTREA_AGENT_BIN}
-    else
-        download_file "${AGENT_BIN}" "${tmp_dir}"/${ANTREA_AGENT_BIN}
-    fi
+    download_bin "${AGENT_BIN}" "${tmp_dir}"/${ANTREA_AGENT_BIN}
```

Comment:
You can cherry pick the commit from my PR.
https://github.com/antrea-io/nephe/pull/52